src/tools/pontoon/users.md

```diff
@@ -4,65 +4,96 @@
 
 ## Creating an account
 
-Pontoon currently uses Firefox Accounts to manage users, so you will need to [create an account](https://accounts.firefox.com/signup) in order to log in.
+While Pontoon can be configured to use [different login systems](https://mozilla-pontoon.readthedocs.io/en/latest/admin/deployment.html#environment-variables) (Django, GitHub, etc.), the Mozilla instance hosted at [pontoon.mozilla.org](https://pontoon.mozilla.org/) uses Firefox accounts.
 
-To create an account on [Pontoon](https://pontoon.mozilla.org/), simply click on the *Sign In* button in the top right corner.
-
-![Pontoon login](../../assets/images/pontoon/users/menu_login.png)
+To create an account, click on the `Sign In` button in the top right corner and follow the instructions.
 
 ## User settings
 
-[User settings](https://pontoon.mozilla.org/settings/) can be accessed by clicking the user avatar in the top right corner, and selecting *Settings*.
+User settings can be accessed by:
+* Clicking the user avatar in the top right corner, and selecting `Settings` from the dropdown menu.
+* Opening directly the `/settings` URL (e.g. [pontoon.mozilla.org/settings](https://pontoon.mozilla.org/settings/)).
+* Using the `CHANGE SETTINGS` button in the [Profile page](profile.md).
 
-![Pontoon login](../../assets/images/pontoon/users/menu_settings.png)
+![User menu with profile settings](../../assets/images/pontoon/users/profile_settings.png "User menu with profile settings")
 
-Profile picture can be set or updated on [Gravatar](http://gravatar.com/) by clicking directly on the image.
+### Profile picture
 
-In the *Personal information* section it’s possible to update the displayed name and change the email address used to log in to Pontoon.
+The profile picture can be set or updated on [Gravatar](http://gravatar.com/) by clicking the image displayed at the top of the settings page (when hovered, the picture will display an `Update profile picture` message).
```

Comment:
```suggestion
The profile picture can be set or updated on [Gravatar](http://gravatar.com/) by clicking the image displayed at the top of the settings page (when hovered, the picture will display a `Change your profile picture` message).
```

Reply:
That's not what the message is in Pontoon?

Reply:
You're right, I was looking at the option to change within my profile not settings, which has a different string.
![Screenshot 2023-05-05 at 09 50 25](https://user-images.githubusercontent.com/38174554/236518961-10121937-9ba8-454a-8e5f-d6da9d0cd9aa.png)

Reply:
Updated issue to make consistent as "Update profile picture", so no change here.